src/main/java/org/folio/marc4ld/service/marc2ld/field/property/transformer/PropertyTransformerFactoryImpl.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.marc4ld.service.marc2ld.field.property.transformer;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.marc4ld.configuration.property.Marc4BibframeRules;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class PropertyTransformerFactoryImpl implements PropertyTransformerFactory {
+
+  private final SeparatePropertyTransformer separatePropertyTransformer;
+  private final DefaultPropertyTransformer defaultPropertyTransformer;
+
+  @Override
+  public PropertyTransformer get(Marc4BibframeRules.FieldRule rule) {
+    if (rule.getTypes().contains("PLACE")) {
```

Comment:
Here also, no special property needed for "Place" alone.

Reply:
Discussed, isolate in factory to understand all the cases